plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/NJSScanReader.java

```diff
@@ -213,34 +213,16 @@ private TestCaseResult produceTestCaseResult(JSONObject file, int cwe_identifier
         return tcr;
     }
 
-    private int cweLookup(int cwe) {
+    private CweNumber cweLookup(int cwe) {
         switch (cwe) {
             case 23: // Relative Path Traversal <-- care about this one
-                return 22; // We expect 22, not 23
-
-            case 79: // XSS <-- care about this one
-            case 209: // Info leak from Error Message
-            case 400: // Uncontrolled Resource Consumption
-            case 522: // Insufficiently protected credentials
-            case 613: // Insufficient session expiration
-            case 614: // Sensitive cookie without Secure Attribute <-- care about this one
-            case 693: // Protection Mechanism Failure (e.g., One or more Security Response header is
-                // explicitly disabled in Helmet)
-            case 798: // Hard coded credentials
-            case 1275: // Sensitive cookie w/ Improper SameSite Attribute
-                break; // Don't care about these, or mapping is correct, so return 'as is'.
-
+                return CweNumber.PATH_TRAVERSAL; // We expect 22, not 23
             case 943: // Improper Neutralization of Special Elements in Data Query Logic (Child of
                 // SQL Injection)
-                return 89; // This is likely an SQLi finding, so mapping to that.
-
-            default:
-                System.out.println(
-                        "WARNING: NJSScan-Unrecognized cwe: "
-                                + cwe
-                                + ". Verify mapping is correct and add mapping to NJSScanReader.");
+                return CweNumber
+                        .SQL_INJECTION; // This is likely an SQLi finding, so mapping to that.
         }
 
-        return cwe;
+        return CweNumber.lookup(cwe);
```

Comment:
Can we restore what we had so we can flag reported CWEs not yet mapped?

Reply:
As discussed: https://github.com/OWASP-Benchmark/BenchmarkUtils/pull/31#issuecomment-1304863379